libnmstate/schema.py

```diff
@@ -247,3 +249,18 @@ class OVSBridge(object):
     PORT_NAME = 'name'
     PORT_VLAN_MODE = 'vlan-mode'
     PORT_ACCESS_TAG = 'access-tag'
+
+
+class Team(object):
+    TYPE = InterfaceType.TEAM
+    CONFIG_SUBTREE = InterfaceType.TEAM
+
+    PORTS_SUBTREE = 'ports'
+    RUNNER_SUBTREE = 'runner'
+
+    class Ports(object):
+        NAME = 'name'
+
+    class Runner(object):
+        NAME = 'name'
+        LOAD_BALANCE = 'loadbalance'
```

Comment:
Since it is one of several possible values, it might make more sense to move this into a `TeamRunner` class or similar like we do it for interface types. It is hard to say with this kind of partially defined schema IMHO. What do you think?

Reply:
You are right, do you like something like `RunnerMode`?

Reply:
Done!